build.gradle

```diff
@@ -216,7 +216,7 @@ def versions = [
         pact            : '4.1.7',
         junitJupiter    : '5.7.2',
         lombok          : '1.18.22',
-        sscsCommon      : '4.6.43',
+        sscsCommon      : '4.6.45-RC-SSCS-10222-1',
```

Comment:
just a question: are we ok to reference the release candidate version of common rather than a stable one?

Reply:
It's not merged into master yet once it is then I will update to an RC version